packages/babel-plugin-transform-classes/test/fixtures/spec/method-check-new-super/expected.js

```diff
@@ -0,0 +1,19 @@
+var Test =
+/*#__PURE__*/
+function (_Foo) {
+  babelHelpers.inherits(Test, _Foo);
+
+  function Test() {
+    babelHelpers.classCallCheck(this, Test);
+    return babelHelpers.possibleConstructorReturn(this, (Test.__proto__ || Object.getPrototypeOf(Test)).call(this));
+  }
+
+  babelHelpers.createClass(Test, [{
+    key: "test",
+    value: function _test() {
```

Comment:
This is introducing a regression here. Currently the generated code is

```
  _createClass(Test, [{
    key: "test",
    value: function test() {
```
and the name is fully preserved, whereas your snippet will name the function `_test`.